src/controllers/queries/__tests__/index.test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+import sinon from 'sinon'
+import { QueriesController } from '../index.js'
+import { withQueriesMocks } from './helpers.js'
+
+describe('QueriesController', () => {
+  afterEach(() => {
+    sinon.restore()
+  })
+  describe('queries', () => {
+    it('should match the snapshot of the rendered query page', async () => {
+      const { mockLogger, templateMock } = withQueriesMocks()
+      const controller = new QueriesController(templateMock, mockLogger)
+
+      const result = await controller.queries()
+
+      expect(result).to.matchSnapshot()
```

Comment:
Please don't use snapshot testing outside of the views. Set up an appropriate templayte mock and use that. You should be able to follow examples elewhere but let me know if this needs clarity